dashboard/app/main.go

```diff
@@ -1229,48 +1237,70 @@ func makeUIJob(job *Job, jobKey *db.Key, bug *Bug, crash *Crash, build *Build) *
 	return ui
 }
 
-func fetchErrorLogs(c context.Context) ([]byte, error) {
+func fetchErrorLogs() ([]byte, error) {
+	if !appengine.IsAppEngine() {
+		return nil, nil
+	}
+
 	const (
-		minLogLevel  = 3
-		maxLines     = 100
-		maxLineLen   = 1000
-		reportPeriod = 7 * 24 * time.Hour
+		maxLines   = 100
+		maxLineLen = 1000
 	)
-	q := &log.Query{
-		StartTime:     time.Now().Add(-reportPeriod),
-		AppLogs:       true,
-		ApplyMinLevel: true,
-		MinLevel:      minLogLevel,
+
+	ctx := context.Background()
```

Comment:
Is there any issue with using the parent context as we do now? Overall it's the right thing to do. This does not happen in background.

Reply:
Fixed.